pkg/skaffold/initializer/deploy/init.go

```diff
@@ -21,8 +21,8 @@ import (
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 )
 
-// deploymentInitializer detects a deployment type and is able to extract image names from it
-type DeploymentInitializer interface {
+// Initializer detects a deployment type and is able to extract image names from it
+type Initializer interface {
```

Comment:
Why?

Reply:
it reads a little clearer now, since we're using `build.Initializer` and `deploy.Initializer` in the same code block